extractors/README.md

```diff
@@ -0,0 +1,45 @@
+# codeql-sap-js:extractors/README.md
+
+## CodeQL CDS Extractor : Flowchart
+
+The following flowchart shows the flow of execution for the current implementation of the extractor.
+
+```mermaid
+flowchart TD
+    COM["`export _build_cmd=<br>$(pwd)/extractors/
+javascript/tools/
+pre-finalize.sh`"]
+    DCR[codeql database create<br>--command=$_build_cmd<br>--language=javascript<br>--search-path=./extractors/<br>--<br>/path/to/database]
+    DB@{ shape: cyl, label: "/path/to/database" }
+    DINIT[codeql database init]
+    CRE[codeql resolve extractor]
+    DTRAC[codeql database<br>trace-command]
+    SPF[[pre-finalize.sh]]
+    JSE[[javascript extractor]]
+    SIF[[index-files.sh]]
+    DIDX[codeql database index-files<br> --language=cds<br>--include-extension=.cds]
+    CC[[\`cds\` compiler]]
```

Comment:
I'm seeing `Unsupported markdown: escapecdsUnsupported markdown: escape compiler` in the preview in GitHub for this node.